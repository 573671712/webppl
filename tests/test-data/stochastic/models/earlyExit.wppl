var model = function() {
  var steps0 = 0;
  var i = randomInteger(10);
  factor(i === 5 ? 0 : -Infinity)

  var steps1 = steps0 + 1;

  var j = randomInteger(12);
  factor(j === 6 ? 0 : -Infinity)

  var steps2 = steps1 + 1;

  return [steps2,i,j];
};
