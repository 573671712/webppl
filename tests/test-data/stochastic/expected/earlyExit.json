{
  "hist": {
    "[2,5,6]": 1
  }
}
